portfolio/src/main/webapp/style.css

```diff
@@ -258,25 +246,19 @@ html {
     border-width: 0 5px;
     padding: 50px;
     width: 300px;
-    top: -600px;
+    top: 205px;
     height: auto;
+    padding-bottom: -50%;
 }
 
 .proj-3-container:hover {
-    position: relative;
-    text-align: center;
-    margin-left: 20%;
     background-color: rgba(151, 191, 128, 0.877);
-    padding: 50px;
-    width: 300px;
-    top: -600px;
-    height: auto;
 }
 
 .proj-4-container {
     position: relative;
     text-align: center;
-    margin-left: 50%;
+    margin-left: 50.6%;
```

Comment:
These are pretty odd percents,  does the 0.6% actually make a difference?

Reply:
Yeah, 50% for project 4 is not perfectly in line with 50% for project 2. For some reason the hover and on click for the bottom two projects didn't work when their position was set to be absolute. However, I had to change the position to be absolute for the top two projects and adjust their alignment in order to fix the padding issues in my website. When I changed the bottom two to relative position, the alignment was really weird so I had to adjust the percentages.